### PR TITLE
feat: `graph-ts` docs

### LINKED ISSUE #429: Integrate Graph TS docs
https://github.com/graphprotocol/graph-tooling/tree/main/packages/ts

### PR DESCRIPTION
**Pull request by saihaj**
CleanShot 2023-11-16 at 10 54 07

closes #429

### CODE REVIEW

File: website/next.config.js [hunk 1]
```diff
@@ -61,6 +61,21 @@ export default withNextra({
...
+    {
+      source: '/en/developer/assemblyscript-api/',
+      destination: '/en/developing/graph-ts/api/',
+      permanent: true,
+    },
+    {
+      source: '/en/developing/assemblyscript-api/',
+      destination: '/en/developing/graph-ts/api/',
+      permanent: true,
+    },
```
Comment (benface) [inline website/next.config.js:78]: Thank you for setting these up!

File: website/pages/en/developing/graph-ts/[[...slug]].mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { RemoteContent } from 'nextra/data'
+import { buildDynamicMDX } from 'nextra/remote'
+import { getPageMap } from '@/src/getPageMap'
+import { remarkReplaceLinks } from '@/src/remarkReplaceLinks'
+import { Mermaid } from 'nextra/components'
```
Comment (benface) [inline website/pages/en/developing/graph-ts/[[...slug]].mdx:5]: 🧜‍♀️

**Review by benface — APPROVED** [review]
🚀

**Comment by azf20** [comment]
Nice! I think my only thought here is that it's `graph-ts`, but it's actually Assemblyscript 🤔

**Pull request merged**